models/refugee.py

```diff
@@ -0,0 +1,101 @@
+from datetime import date, datetime
+from enum import Enum
+
+class Refugee:
+    """
+    A class to represent a refugee family.
+    """
+    class MedicalCondition(Enum):
+        MINOR = "minor"
+        MAJOR = "major"
+        MODERATE = "moderate"
+        EXTREME = "extreme"
+
+    def __init__(self,
+                 firstname: str,
+                 lastname:str,
+                 camp: str,
+                 medicalConditionType: MedicalCondition,
+                 numOfFamilyMember: int,
+                 startingDate=datetime.today().date()):
+
+        """
+        :param firstname: firstname of refugee family
+        :param lastname: lastname of refugee family
+        :param camp: camp of the refugee family locating at
+        :param medicalConditionType: medical condition type of the refugee
+        :param numOfFamilyMember: param number of family member
+        :param startingDate: start date of refugee family creation
+        """
+
+        self.name = self.__checkName(firstname, lastname)
+        self.numOfFamilyMember = self.__checkNumOfFamilyMember(numOfFamilyMember)
+        self.camp = camp
+        self.medicalConditionType = medicalConditionType
+        self.startingDate = self.__checkStartingDate(startingDate)
```

Comment:
I still prefer the methods to not be named check_xxxx if they return a value. 
Maybe sth like `sanitise_name`?